services/credential-server/src/agent.ts

```diff
@@ -264,6 +264,20 @@ class Agent {
     );
     await this.signifyApi.deleteNotification(grantNotification.i);
   }
+
+  async saveSchema(schema: any, label?: string): Promise<void> {
+    const { saidifiedSchema, customizableKeys } =
+      await this.signifyApi.saidifySchema(schema, label);
+    let schemas = await lmdb.get(SCHEMAS_KEY);
```

Comment:
better to do something like the key as `schema:<schemaId>` and then store them individually. Rather than a single big object for all schemas.